packages/ListBoxBrowser/src/components/OptionsSelected/OptionsSelected.js

```diff
@@ -0,0 +1,96 @@
+import React from "react";
+import RawButton from "@paprika/raw-button";
+import Times from "@paprika/icon/lib/Times";
+import useI18n from "@paprika/l10n/lib/useI18n";
+import { getBreadcrumb } from "../../helpers";
+import { ListBoxBrowserContext } from "../../ListBoxBrowser";
+
+import {
+  optionStyles,
+  container,
+  title,
+  button,
+  remove,
+  counter,
+  optionLabel,
+  optionBreadcrum,
+  noOptionsSelectedContainer,
+  noOptionsSelected,
+} from "./OptionsSelected.styles";
+
+function getLength(options) {
+  let i = 0;
```

Comment:
perhaps use lodash size function instead of this?

Reply:
I would prefer to have a small function and avoid the dependency :)

Reply:
This seems more complicated than size though because it counts the lengths of the arrays in the values?
The implementation looks a bit off to me: I don't think the inner `forEach` and the return are necessary (`forEach` itself returns undefined anyway).

```js
Object.values(options).reduce((acc, option) => acc + option.length, 0);
```

Maybe also rename the function to something like `getTotalOptionsLength`?